staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1926,11 +1926,12 @@ type DeveloperConfiguration struct {
 // LogVerbosity sets log verbosity level of  various components
 // +k8s:openapi-gen=true
 type LogVerbosity struct {
-	VirtAPI        uint `json:"virtAPI,omitempty"`
-	VirtController uint `json:"virtController,omitempty"`
-	VirtHandler    uint `json:"virtHandler,omitempty"`
-	VirtLauncher   uint `json:"virtLauncher,omitempty"`
-	VirtOperator   uint `json:"virtOperator,omitempty"`
+	VirtAPI         uint `json:"virtAPI,omitempty"`
+	VirtController  uint `json:"virtController,omitempty"`
+	VirtHandler     uint `json:"virtHandler,omitempty"`
+	VirtLauncher    uint `json:"virtLauncher,omitempty"`
+	VirtOperator    uint `json:"virtOperator,omitempty"`
+	LibvirtLogLevel int  `json:"libvirtLogLevel,omitempty"`
```

Comment:
Maybe it's better to define it as `uint` like the other struct's attributes

Reply:
Also - the fact that all of these attributes represent Kubevirt components - makes me a bit uncomfortable for a few reasons:

1. The user, who's expected to be familiar with Kubevirt at a certain level, knows about our different components (virt-launcher, handler, etc') but does not necessarily know about libvirt.
2. The fact that we expose our core technology in our configurations couples us to that technology in some sense.

I think it may be reasonable to simply enable libvirt logging when `VirtLauncher` logging is enabled. That's sort of what happens today IIUC - when `VirtLauncher` logging is requested we enable logs from QEMU, etc' without the user needing to specify this explicitly.

All of that being said - we're talking about `DeveloperConfiguration`, which implies that these are "advanced" configuration for advanced users, so I do think it's reasonable.

WDYT?

Reply:
Thank you for the review, it does make a lot of sense.
Closing for now, this needs to be rethought.